File: ASPNETCore/HowTo/WebApiTypescriptSample/src/Modules/Excel/Dtos/ReplaceText.dto.ts
import { BaseDto } from "../../../Base/Base.Dto";

export class ReplaceTextDto extends BaseDto {

    /**The excel file name that storage manager can recognize. */
    public ExcelPath: string;

    /**The sheet name (if not provide, replace in all sheets). */
    public SheetName: string;

    /**The text which is replaced. */
    public Text: string;

    /**The new text to replace. */
    public NewText: string;

    /**A boolean value indicates whether to search the value with case sensitive. */
    public MatchCase: boolean;
    
    /**A boolean value indicates whether to search the value with matching a whole word. */
    public WholeCell: boolean;

}

File: ASPNETCore/HowTo/WebApiWPFSample/WebApiWPFSample/src/Modules/Excel/Dtos/ReplaceText.dto.cs
namespace WebApiConsoleSample
{
    public class ReplaceTextDto : BaseDto
    {
        /**The excel file name that storage manager can recognize. */
        public string ExcelPath { set; get; }

        /**The sheet name (if not provide, replace in all sheets). */
        public string SheetName { set; get; }

        /**The text which is replaced. */
        public string Text { set; get; }

        /**The new text to replace. */
        public string NewText { set; get; }

        /**A boolean value indicates whether to search the value with case sensitive. */
        public bool MatchCase { set; get; }
        
        /**A boolean value indicates whether to search the value with matching a whole word. */
        public bool WholeCell { set; get; }

    }
}